kkm/kkm_main.c

```diff
@@ -358,15 +359,15 @@ static long kkm_execution_kontext_ioctl(struct file *file_p,
 					xs->valid;
 				if (xs->format != kkm_xs_format) {
 					printk(KERN_NOTICE
-					       "XSTATE saved format mismatch expecting %x found %x",
+					       "kkm_execution_kontext_ioctl: XSTATE saved format mismatch expecting %x found %x",
```

Comment:
Does the compiler support the predefined symbol __FUNCTION__?
If so you could use __FUNCTION__ as an argument to printk() or prepend to the message string.

Reply:
github is messing with my comment.
__FUNCTION__ should be underbar underbar FUNCTION underbar underbar.

Reply:
It is not generally used in kernel code.

We are outside kernel module, we can write our own wrappers to printk. I chose to keep in sync with kernel code.